core/src/proof/normalization.rs

```diff
@@ -28,3 +32,10 @@ pub fn create_hashed_normalized_doc(
 
     Ok(output)
 }
+
+pub fn hash(data: &[u8]) -> Result<[u8; 32], crate::error::Error> {
```

Comment:
Are any of these functions fallible? Why does this need to be `Result<_, Error>`?

Reply:
Good catch!